src/main/java/com/hubspot/jtopia/entities/ConfigurationIF.java

```diff
@@ -0,0 +1,25 @@
+package com.hubspot.jtopia.entities;
+
+import java.util.Set;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@Value.Immutable
+@Value.Style(
```

Comment:
should we just make a proper style annotation here instead of copying this style information on each of the immutables?